packages/react/src/components/Disclosure/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { useRef, useState } from 'react';
+
+function useDisclosure(id) {
+  const [open, setOpen] = useState(false);
+  const ref = useRef(null);
```

Comment:
Seems like this might not be used in the implementation / test so could be removed